devices/stm32wle5.yaml

```
@@ -0,0 +1,4 @@
+_svd: ../svd/STM32WLE5.svd
```

```suggestion
_svd: ../svd/stm32wle5.svd
```

The extraction script makes all the SVD files lowercase.